demo/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
+import logo from "./logo.svg";
```

Comment:
🛑 Not used? Also it doesn't appear to be added to the PR?

Reply:
🛑 Are our linters working against these files? It doesn't seem like they are, if this slipped through. We should make sure they are!